File: src/main/java/dev/kawrl/botcommands/PingCommand.java
package dev.kawrl.botcommands;

import dev.kawrl.interfaces.SlashCommandInterface;
import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class PingCommand implements SlashCommandInterface {
    private static final Logger log = LoggerFactory.getLogger(PingCommand.class);

    @Override
    public void execute(SlashCommandInteractionEvent event) {
        long ping = event.getJDA().getGatewayPing();
        event.reply(String.format("Pong! 🏓 (%d ms)", ping)).queue();
        log.info("Bot has been pinged by {}", event.getUser().getName());
    }
}

File: src/main/java/dev/kawrl/botcommands/PingCommand.kt
package dev.kawrl.botcommands

import dev.kawrl.interfaces.CommandHandler
import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent

class PingCommand : CommandHandler(), CommandHandler.SlashCommandInterface {
    override fun execute(event: SlashCommandInteractionEvent) {
        val ping = event.jda.gatewayPing
        event.reply("Pong! 🏓 ($ping ms)").queue()
        logger.info("Bot has been pinged by {}",event.user.name)
    }
}